src/js/components/Form/__tests__/Form-test-controlled.tsx

```diff
@@ -756,7 +770,7 @@ describe('Form controlled', () => {
         firstName: 'J',
         middleName: '1',
         lastName: '',
-        title: 1,
+        title: '1',
```

Comment:
so I was looking at this but dont have background on the test I think we can also add 
`      const [title, setTitle] = useState<number | string>(1);`
so that covers us there then for the `.length` we can do something like this 

```

      validate={[
              { regexp: /^[a-z]/i },
              () => {
                if (typeof title === 'string' && title.length === 1)
                  return 'must be >1 character';
                return undefined;
              },
            ]}

```

Reply:
Reverted the title field back to a number and updated the validation accordingly!